doc/tutorials.rst

```diff
@@ -20,37 +60,119 @@ plugins, and devices, via simple and easy-to-follow examples.
 
 .. customgalleryitem::
     :tooltip: Use quantum machine learning to tune a beamsplitter.
-    :figure: ../examples/figures/gaussian_transformation.png
+    :figure: ../examples/figures/gaussiancircuit.png
     :description: :ref:`gaussian_transformation`
 
+:html:`<div style='clear:both'></div>`
+
+Classical interfaces
+~~~~~~~~~~~~~~~~~~~~~
+
+The following tutorials take an in-depth look at one and the same Ising model tutorial, implemented across different interfaces offered by PennyLane. 
+
+.. toctree::
+    :hidden:
+    :maxdepth: 2
+
+    tutorials/pennylane_isingmodel_NumPy 
+    tutorials/pennylane_isingmodel_TF
+    tutorials/pennylane_isingmodel_PyTorch
+
+.. customgalleryitem::
+    :tooltip: Ising model example with PennyLane NumPy interface.
+    :figure: ../examples/figures/pennylane_xanadu.png
+    :description: :ref:`isingmodel_NumPy`
+
+.. customgalleryitem::
+    :tooltip: Ising model example with PennyLane TensorFlow interface.
+    :figure: ../examples/figures/tensorflow.png
+    :description: :ref:`isingmodel_TF`
+
+.. customgalleryitem::
+    :tooltip: Ising model example with PennyLane PyTorch interface.
+    :figure: ../examples/figures/pytorch.png
+    :description: :ref:`isingmodel_PyTorch`
+
+:html:`<div style='clear:both'></div>`
+
+
+Plugins and hybrid computation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The most powerful design feature of PennyLane is that it allows the user to run hybrid computations on a variety of different simulators and real quantum hardware. The following tutorials introduce the notion of hybrid computation by combining several PennyLane plugins.
```

Comment:
Are they really? The "pytorch_noise" tutorial integrates the forest plugin with the pytorch interface, and the "state_prep" one also uses forest only. 

I propose to change the summary a bit, so people don't get confused...